python/py-blis/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                      1.0
+PortGroup           python                      1.0
+PortGroup           github                      1.0
+
+name                py-blis
+version             0.3.0
+revision            0
+github.setup        explosion cython-blis ${version} v
+
+checksums           rmd160  2eb5c8a0e13a8b8fb2095ba9ace21236c130a956 \
+                    sha256  6331325269afd959c57c093bf1c9f02eb6be28cbcacc9279d6de4436464cc90f \
+                    size    1663871
+
+platforms           darwin
+supported_archs     x86_64
+
+license             MIT
+
+maintainers         {jonesc @cjones051073} openmaintainer
+
+description         Fast matrix-multiplication as a self-contained Python library
+long_description    ${description}
+
+github.livecheck.regex  {([0-9.]+)}
+
+# Support python versions
+python.versions     35 36 37
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-wheel
+
+    depends_lib-append \
+        port:py${python.version}-cython \
+        port:py${python.version}-pytest \
+        port:py${python.version}-numpy \
+        port:py${python.version}-hypothesis
+
+    post-extract {
+        # Fix hard-coded dep ...
+        reinplace "s|0.33.0|0.33.2|g" ${worksrcpath}/setup.py
+        # Set cython version
+        set PythonVersionWithDot [join [split ${python.version} ""] "."]
+        reinplace "s|\'cython\'|\'cython-${PythonVersionWithDot}\'|g" ${worksrcpath}/bin/cythonize.py
+    }
+    
+    build.cmd    "${python.bin} setup.py build"
+    destroot.cmd "${python.bin} setup.py install"
```

Comment:
why are you overriding the ```build.cmd``` and ```destroot.cmd``` ; it seems to be fine with the defaults from the Python PortGroup?

Reply:
I guess I did not realise they where the defaults. Thanks, I remove them.

Reply:
I am not sure if these are the exact defaults from the PortGroup (haven't checked....), but when removing these overrides it still builds fine. So whatever the default are in the portgroup they seem to work.

Reply:
yes, I have gone through and removed most cases.